code/__HELPERS/_macros.dm

```diff
@@ -131,6 +133,10 @@
 
 #define isatom(A) (istype(A, /atom))
 
+#define issqueletic(A) (isskellington(A) || isskelevox(A)) //Yes, it is a word
```

Comment:
You should make names as obvious as possible. Nobody will guess issqueletic, isskeletal is much better.